src/components/SimpleInput.js

```diff
@@ -0,0 +1,58 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+const Input = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    type: React.PropTypes.string,
+    valid: React.PropTypes.bool
+  },
+
+  getDefaultProps () {
+    return {
+      type: 'text',
+      valid: true
+    };
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <input
+        {...this.props}
+        name={this.props.name}
+        style={[styles.wrapper, !this.props.valid && styles.error, this.props.style]}
```

Comment:
I'm not seeing `Object.assign` used anywhere.  Did you mean that wanted to leave it as is because of the `:focus`/Radium dependency or did you change it and just not push up the change?